snapshotserver/config.go

```diff
@@ -50,7 +47,16 @@ func SetConfigDefaults() {
 	pflag.StringP("tempdir", "T", "", "Set temporary directory for snapshot files")
 	viper.SetDefault("tempdir", defaultTempDir)
 
-	pflag.BoolP("config", "C", false, fmt.Sprintf("Use a config file named '%s' located in either /etc/%s/, ~/.%s or ./)", appName, packageName, packageName))
+	pflag.IntP("connmaxlife", "x", 5, "Sets the maximum amount of time (Minutes) a connection may be reused")
```

Comment:
Also, it looks like you need to run "go fmt" on your modified code -- right now the presubmit tests are failing because the formatting doesn't match.

You can run "make presubmit" at the top level to run those checks.